app/controllers/application_controller.rb

```diff
@@ -2,7 +2,7 @@ class ApplicationController < ActionController::Base
   http_basic_authenticate_with(
     name: Horizon.config[:basic_auth_user],
     password: Horizon.config[:basic_auth_pass]
-  ) if Horizon.config[:basic_auth_pass].present?
+  ) if !Rails.env.test? && Horizon.config[:basic_auth_pass].present?
```

Comment:
I was thinking about this last night and I wonder if there's a more declarative way to approach this. Like, what if this looked more like this:

```rb
http_basic_authenticate_with(Horizon.config[:basic_auth_creds]) if Horizon.config[:basic_auth_enabled]
```

And then in the test environment we set that `basic_auth_enabled` to false. That seems less mysterious to me but certainly not blocking feedback, just something I was thinking about! It also demotes this controller to knowing less about the way config happens ya know? What do you think?

Reply:
I like it! Adding now.

Reply:
Decided to hold on this for the moment -- this approach requires us setting two env vars rather than one which while more declarative feels extra. I'm hoping we can convert to artsy oauth and remove this feature entirely.

Reply:
Oooo, yeah that's a better idea for sure! 🏆